Makefile

```diff
@@ -0,0 +1,60 @@
+# Makefile for AirCan
+PACKAGE_NAME := aircan
+PACKAGE_DIRS := aircan
+TESTS_DIR := tests
+VERSION_FILE := VERSION
+
+PYTEST_EXTRA_ARGS := --flake8 --isort --doctest-modules -rsxX -l --tb=short --strict --cov-report term-missing:skip-covered --cov=tests --cov-fail-under 100
```

Comment:
Should we maybe stick to a simpler `--flake8 --isort --doctest-modules` without the remaining options? It makes for a nice output, but might be cumbersome if many tests fail at once.